src/SUMMARY.md

```diff
@@ -1,21 +1,28 @@
 # Summary
 
-- [About this guide](./about-this-guide.md)
+[About this guide](./about-this-guide.md)
+
+---
+
+- [Part 1 Intro](./part-1-intro.md)
```

Comment:
I think we should name this something like

Part 1: Building, debugging, and contributing to Rustc

although that seems a bit long to me =)

Reply:
Same.